src/applications/vaos/containers/VAFacilityPageV2.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { scrollAndFocus } from '../utils/scrollAndFocus';
+
+import {
+  openFacilityPage,
+  updateFacilityPageData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../new-appointment/redux/actions';
+import { getFacilityPageInfo } from '../utils/selectors';
+
+const initialSchema = {
+  type: 'object',
+  required: ['vaParent', 'vaFacility'],
```

Comment:
Since this new page flattens the selections, we'll probably need only `vaFacility` and not the `vaParent` in this schema.

Reply:
Removed the references to vaParent